egs/swbd/s5c/local/nnet3/run_ivector_common.sh

```diff
@@ -31,7 +31,7 @@ if [ "$speed_perturb" == "true" ]; then
       utils/fix_data_dir.sh data/${datadir}_tmp
 
       utils/copy_data_dir.sh --spk-prefix sp1.0- --utt-prefix sp1.0- data/${datadir} data/temp0
-      utils/combine_data.sh data/${datadir}_sp data/${datadir}_tmp data/temp0
+      utils/combine_data.sh --extra-files utt2uniq data/${datadir}_sp data/${datadir}_tmp data/temp0
```

Comment:
... well, it should happen automatically if any source directory has the utt2uniq file.
Otherwise there is no need, if we follow the convention that no utt2uniq file means the utterance-id itself is the utt2uniq file, because having no utt2uniq file gives us the correct mapping.